### PR TITLE
Add command for task metrics

### PR DESCRIPTION
**Pull request by philipnrmn**
This PR adds the `dcos task metrics` command. 

Components:
- [x] `dcos task metrics details`
- [x] query `/app` endpoint
- [x] `dcos task metrics summary`
- [x] update task metrics to take account of https://github.com/dcos/dcos-metrics/pull/64
- [x] truncate tags in `dcos task metrics details`
- [x] integration tests
- [x] remove integration tests
- [x] substitute unit tests for integration tests

Example usage:

```bash
$ dcos task metrics details app-name.abc123-456789
NAME                       VALUE        
cpus.user.time             0.14         
cpus.system.time           0.21         
cpus.limit                 0.10         
cpus.throttled.time        471.89       
mem.total                  0.01GiB      
mem.limit                  0.16GiB      
disk.limit                 0.00GiB      
disk.used                  0.00GiB      
net.rx.packets             0            
net.rx.bytes               0.00GiB      
net.rx.errors              0            
net.rx.dropped             0            
net.tx.packets             0            
net.tx.bytes               0.00GiB      
net.tx.errors              0            
net.tx.dropped             0            
statsd_tester.time.uptime  11632203.00
```

```bash
$ dcos task metrics details app-name.abc123-456789 --json
[
  {
    "name": "cpus.user.time",
    "tags": {...},
    "timestamp": "2017-03-15T21:01:46.188487097Z",
    "unit": "seconds",
    "value": 0.14
  },
  ...
  {
    "name": "statsd_tester.time.uptime",
    "timestamp": "2017-03-15T21:02:40Z",
    "unit": "",
    "value": 11672303.0
  }
]
```

```bash
$ dcos task metrics summary app-name.abc123-456789
CPU           MEM              DISK             
0.35 (0.07%)  0.01GiB (5.41%)  0.00GiB (0.00%)  
```

```bash
$ dcos task metrics summary app-name.abc123-456789 --json
[
  {
    "timestamp": "2017-03-15T21:03:46.221983351Z",
    "unit": "seconds",
    "value": 0.14,
    "name": "cpus.user.time",
    "tags": {...}
  },
  {
    "timestamp": "2017-03-15T21:03:46.221983351Z",
    "unit": "bytes",
    "value": 9080832,
    "name": "mem.total",
    "tags": {...}
  },
  {
    "timestamp": "2017-03-15T21:03:46.221983351Z",
    "unit": "bytes",
    "value": 0,
    "name": "disk.used",
    "tags": {...}
  }
]

```

### CODE REVIEW

**Comment by philipnrmn** [comment]
@tamarrow and @malnick, may I have a review when you get a moment?

File: cli/dcoscli/task/main.py [hunk 1]
```diff
@@ -550,3 +562,33 @@ def _load_slaves_state(slaves):
...
+    :param task_id: mesos task id
+    :type task_id: str
+    :param json: print raw JSON
+    :type json: bool
+    :return: Process status
+    :rtype: int
+    """
+
+    master = mesos.get_master()
+    slave_id = master.task(task_id)['slave_id']
```
Comment (tamarrow) [inline cli/dcoscli/task/main.py]: what if there are no slaves?

**Comment by tamarrow** [comment]
Looks great. Is there a structure to the tags or can they be anything?

**Comment by malnick** [comment]
Tags are arbitrary stringified key, value pairs.

File: cli/dcoscli/task/main.py (hunk 1 above)
Comment (philipnrmn) [inline cli/dcoscli/task/main.py]: I've added a guard for this, but I don't think a Mesos task object can ever not have a slave_id field.

**Pull request merged**